.jenkins/short-perf-test-cpu.sh

```diff
@@ -8,10 +8,37 @@ export PATH=/opt/conda/bin:$PATH
 
 echo "Running CPU perf test for PyTorch..."
 
+# Get last master commit hash
+export PYTORCH_COMMIT_ID=$(git log --format="%H" -n 1)
+
+# Get baseline file from https://github.com/yf225/perf-tests
+cp /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json perf_test_numbers_cpu.json
```

Comment:
Doesn't have to be for this PR, but let's be mindful of people who will want to run these perf test scripts directly when they want to reproduce CI results. Hardcoded paths like this make the script less portable.